cli/azd/test/functional/testdata/samples/springapp/azure.yaml

```diff
@@ -0,0 +1,13 @@
+name: webapp
+metadata: 
+  template: azd-test/webapptest@v1
+services:
+  sweb:
+    project: ./
+    host: springapp
+    language: java
+    spring:
```

Comment:
We will want to add information about this object (and the deploymentName property) to our `azure.yaml` schema before GAing this feature.  Let's open a work item to track doing that.